File: file_write/csharp_file_write/file_write.cs
using System;
using System.IO;

public class FileWrite {
  public static void Main( string[] args ) {
    if( args.Length != 0 ) {
      Console.WriteLine("Usage: " + Environment.GetCommandLineArgs()[0] + " <fname>");
      Environment.Exit(1);
    }
    //file_write( args[0], "mydata" );
    file_write("output.txt", "hello world");
  }

  public static void file_write( string fname, string data ) {
    //string fname = string.Empty;
    StreamWriter streamWriter = null;
    FileStream fileStream = null;

    try {
      fileStream = new FileStream(fname, FileMode.Create, FileAccess.Write);
      streamWriter = new StreamWriter(fileStream);
      streamWriter.Write(data);
      streamWriter.Close();
      fileStream.Close();
    }

    catch( Exception e ) {
      Console.WriteLine( e.Message );
    }
  }
}

File: file_write/fsharp_file_write/file_write.fs
#light
open System
open System.IO

//comment
(* commment *)

(*

let file_write fileName =
    use streamWriter = new StreamWriter("output.txt", false)
    streamWriter.WriteLine("<startTag>")
    seqLen |> xmlSeq |> Seq.iter streamWriter.WriteLine
    streamWriter.WriteLine("</startTag>")

let wr = new System.IO.StreamWriter("output.csv")
table |> List.map(string) |> String.concat("") |> wr.Write
wr.Close()
*)

[<EntryPoint>]
let main args =
  let argv = Environment.GetCommandLineArgs()

  if argv.Length <> 1 then
    eprintfn "Usage: %s <noargs>" argv.[0]
    -1
  else
    File.WriteAllText("output.txt", "Hello World")
    printfn "wrote output.txt"
    0